HW1/CreditReport.java

```diff
@@ -0,0 +1,130 @@
+import java.util.Scanner;
+class CreditReport {
```

Comment:
please rename all files in HW1 with your initials,,,,, will CREATE CONFLICTS WHEN I MERGE WITH TUM'S!